src/helpers/user_normalizer.coffee

```diff
@@ -0,0 +1,7 @@
+class UserNormalizer
+  @normalize: (username, userInput)->
+    if not userInput? or (userInput?.toLocaleLowerCase() is 'me')
+      return '@' + username
+    '@' + userInput.replace /@*/g, ''
```

Comment:
We use Slack in our company, so when people are adding other people to the team they are used to use the autocomplete for @ mentions. Some people use `hubot team +1`, the other ones use `hubot team add @username` after a few user been added to the team, the list will be something like this:

```
Team:
1. @username1
2. username2
```

This is why we added a normalizer for usernames, we can make it configurable through an env var if you prefer.